client/coral-embed-stream/src/Embed.js

```diff
@@ -158,6 +159,11 @@ class Embed extends Component {
               clearNotification={this.props.clearNotification}
               notification={{text: null}}
             />
+          <LoadMore
+            asset_id={asset.id}
+            comments={asset.comments}
+            moreComments={asset.commentCount > asset.comments.length}
```

Comment:
This is a bit confusing, but stems from a product decision. The call was only to include top level comments in the commentCount we show in the tab and not count children.

So, the variable name is misleading, but the confusion stems from our comment counting methodology.